locale/action_names.rb

```diff
@@ -1,5 +1,5 @@
 # Autogenerated!
-_("Action with sub plans")
-_("Import Puppet classes")
-_("Import facts")
 _("Remote action:")
+_("»Action with sub plans«")
+_("»Import Puppet classes«")
+_("»Import facts«")
```

Comment:
Why do these get `»` and `«`? I think it's the "mark for translation" setting that adds this, but is it leading to double translations?

Reply:
Interesting, it wasn't there before I updated `foreman-tasks` plugin. I will fix it